app-devel/geany/autobuild/prepare

```diff
@@ -0,0 +1,2 @@
+abinfo "Setting python intepreter to python3 ..."
+export PYTHON=python3
```

Comment:
You might want to set this variable with full path, as they used to use this variable to fill in their shebang (and this won't work).

Reply:
Fixed